frontends/election-manager/src/screens/tally_report_screen.tsx

```diff
@@ -234,9 +254,19 @@ export function TallyReportScreen(): JSX.Element {
               </Button>
             )}
           </p>
+          {location.pathname === '/reports/full' && (
```

Comment:
Let's use `routerPaths.tallyFullReport` here, just in case we ever change the path